src/bootstrap/util.rs

```diff
@@ -479,17 +479,17 @@ pub fn get_clang_cl_resource_dir(clang_cl_path: &str) -> PathBuf {
     clang_rt_dir.to_path_buf()
 }
 
-pub fn lld_flag_no_threads(is_windows: bool) -> &'static str {
```

Comment:
Not sure why this function was split into two.
`fn is_lld_newer_than_10` has only a single call site now and can be inlined and removed.